flows/actions/call_webhook.go

```diff
@@ -92,6 +93,5 @@ func (a *CallWebhookAction) Execute(run flows.FlowRun, step flows.Step, log flow
 		log.Add(events.NewWebhookCalledEvent(webhook.URL(), webhook.Status(), webhook.StatusCode(), webhook.Request(), webhook.Response()))
 	}
 
-	run.SetWebhook(webhook)
```

Comment:
its supposed to be the responsibility of events to update the session state so this moves to the event